### ISSUE TITLE
HTTPS CDN request

### ISSUE THREAD

**Issue by jolexa** [arturadib/strapdown#77]
Hello,
It would be grate to submit this to cdnjs or similar to get https support. The current github cdn does not support https urls.

Thanks for considering

**Comment by gregsadetsky** [comment]
Hopefully this is useful (I'm not affiliated, etc.). You can use:

<https://cdn.rawgit.com/arturadib/strapdown/gh-pages/v/0.2/strapdown.js>

**Comment by ace0** [comment]
Agree: this package looks great, but is useless to me without an HTTPS endpoint.

The suggested link by @gregsadetsky works to load most files, but only the themes listed in:
https://github.com/arturadib/strapdown/tree/gh-pages/v/0.2/themes
are supported.

**Comment by qgustavor** [comment]
> The current github cdn does not support https urls.

[GitHub Pages now supports HTTPS](https://blog.github.com/2018-05-01-github-pages-custom-domains-https/), but to make it work someone needs to fix the DNS records.

**Comment by fu-sen** [comment]
Years have passed. This comment of mine is 2021.

Currently [jsDelivr](https://www.jsdelivr.com/) is the right CDN for that. You can use it now:
```
<script src="https://cdn.jsdelivr.net/gh/arturadib/strapdown@gh-pages/v/0.2/strapdown.min.js"></script>
```

Also, I have the following projects:
https://github.com/fu-sen/strapdown.js
